### ISSUE TITLE
Special thanks and gratitude and the absurd unacceptable revolting unjust sacnadlaous decision of the fiefox store The security of this module is not controlled by Mozilla. Make sure it is safe before installing it and the importance of the child process and sponsor

### ISSUE THREAD

**Issue by trimechee** [andy-portmen/open-in-vlc#62]
Hello, I have been testing since yesterday our beloved Ope-in-vlc for firefox on complex platforms like vidmoly because it does not work on chromium, and I did not like using firefox because we had to keep the child process and not close firefox otherwise the video in vlc closes, but I am testing and we can playback vidmoly videos and other complicated platforms in vlc unlike chrome!!

So Open-in-vlc is really and honestly the most useful vital life saver ecological extension in the world and allows to save millions of machines from destruction and enjoy entertainment or academic videos with old laptop low cpu, thank you very much Mr @andy-portmen :) 💯 🥇 Open-in-vlc is the only extension capable of doing that, even preview video of cat catch does not work with vidmoly in firefox and chrome .....

and I discover with anger disappointment revolt that the best extension in the world and more useful indispensable our so beoved Open-in-vlc, in the firefox stre, is the victim of an unfair mention of firefox store : "

The security of this module is not controlled by Mozilla. Make sure it is reliable before installing it "

I do not understand the firefox stre, Mr Andy is known, famous, has the generosity of offer us its extensions in open source .... it is not the fault of Open-in-vlc if firefox store has a limited team and did not want to review Open-in-vlc, on the contrary mozilla must award the prize for the best recommended extension to Open-in-vlc! I think it would be good to ask Open-in-vlc users to send mails or protest messages to mozilla to remove this unfair mention and put more evidence or pop-up windows to encourage people to help develop this wonderful project by making donations, and I think that the child process is important, i.e. you should not close your web browser when watching vlc player because in chrome, the child process is killed which probably causes the bug with vidmoly and other platforms, there is no way to keep child process in chrome so that the version of Open-in-vlc for chrome is as efficient as the open-in-vlc version for firefox please?

if people don't send enough donations, I think it's a good idea to add an option "display some non-intrusive ads", it will help to better support this wonderful project, thank you very much!

I specify that the famous extension stream recorder doubly certified by google:

https://www.hlsloader.com/

displays some ads in the download page, I saw it with my own eyes, so it's ethical and normal to display some ads if there are not enough paypal or buy me coffee or crypto donations to support the developer....

**Comment by andy-portmen** [comment]
> The security of this module is not controlled by Mozilla. Make sure it is reliable before installing it "

Firefox maintains a recommended list of extensions that undergo thorough review before being published. Extensions not on this list receive less rigorous review. As a result, you'll notice this banner for all extensions except those on the list.

**Comment by trimechee** [comment]
Yes it is incomprehensible big strategic error that mozilla does not integrate Open-in-vlc in the short list of the best extensions of firefox store @mozilla @firefox !

Mozilla should put our beloved Open-in-vlc in a short list of the best extensions because Open-in-vlc is what would motivate the people and free people to prefer firefox against chrome because Open-in-vlc for firefox is more efficient than Open-in-vlc for chrome, tests as proof!
